docs/enterprise/reference/manage.md

```diff
@@ -0,0 +1,369 @@
+---
+title: Manage
+lang: en-US
+sidebarDepth: 2
+meta:
+    - name: keywords
+      content: configuration options settings Pomerium enterprise console
+---
+
+# Manage
+
+## Routes
+
+A Route provides access to a service through Pomerium.
```

Comment:
```suggestion
A Route defines how to access a service running behind Pomerium. This includes authentication (both for Pomerium and passed through to the service), rewrites, header management, load balancing, etc.
```

Reply:
Note that this content is created by `console-settings.yaml` so we'll want to update the copy there, not here.

Reply:
SGTM.